roles/migrationcontroller/templates/velero.yml.j2

```diff
@@ -186,8 +186,10 @@ spec:
       labels:
         name: restic
     spec:
-{% if daemonset_node_selector is defined %}
-      nodeSelector: {{ daemonset_node_selector }}
+{% if _migration_openshift_io_migrationcontroller_spec.daemonset_node_selector is defined %}
+      nodeSelector: {{ _migration_openshift_io_migrationcontroller_spec.daemonset_node_selector }}
```

Comment:
@jmontleon can we use `to_yaml` filter to avoid funny behaviour on bad input? without that, I think the var is passed as json?

Reply:
@pranavgaikwad updated and tested. Looks like it's working so no objection from me!